ocs_ci/ocs/ui/views.py

```diff
@@ -453,6 +454,80 @@
         By.XPATH,
     ),
     "cluster-set-selection": ("//a[normalize-space()='{}']", By.XPATH),
+    "cc_create_cluster": ("createCluster", By.ID),
+    "cc_provider_vmware_vsphere": ("//*[@id='vmware-vsphere']", By.XPATH),
+    "cc_cluster_name": ("input[placeholder='Enter cluster name']", By.CSS_SELECTOR),
+    "cc_base_dns_domain": (
+        "input[placeholder='Enter base DNS domain'",
+        By.CSS_SELECTOR,
+    ),
+    "cc_openshift_release_image": (
+        "input[placeholder='Select or enter a release image']",
+        By.CSS_SELECTOR,
+    ),
+    "cc_vsphere_network_name": (
+        "input[placeholder='vSphere network name']",
+        By.CSS_SELECTOR,
+    ),
+    "cc_api_vip": ("input[placeholder='Enter api vip']", By.CSS_SELECTOR),
+    "cc_ingress_vip": ("input[placeholder='Enter ingress VIP'", By.CSS_SELECTOR),
+    "cc_next_page_button": ("//button[normalize-space()='Next']", By.XPATH),
+    "cc_create_button": ("//button[normalize-space()='Create']", By.XPATH),
+    "cc_cluster_details": ("//div[contains(text(),'Cluster details')]", By.XPATH),
+    "cc_node_pools": ("//div[contains(text(),'Node pools')]", By.XPATH),
+    "cc_networks": ("//div[contains(text(),'Networks')]", By.XPATH),
+    "cc_proxy": ("//div[contains(text(),'Proxy')]", By.XPATH),
+    "cc_review": ("//button[normalize-space()='Review']", By.XPATH),
+    "cc_infrastructure_provider_creds": (
+        "input[placeholder='Select a credential']",
```

Comment:
How will credentials be passed into the XPath?

Reply:
So, the Idea is, by default its a dropdown, I would have created a vsphere cred secret from CLI, then click on this dropdown and select that vsphere cred (whose xpath needs to be dynamically generated at run time).

Reply:
exactly, so you should use {} in the place of `Select a credential` and then pass the cred at run time using the format_locator function we have in helpers_ui file

Reply:
done